extension/manifest.json

```diff
@@ -30,12 +30,12 @@
     "tabs",
     "https://flowcrypt.com/*",
     "https://*.google.com/*",
+    "https://*.googleapis.com/*",
     "https://outlook.live.com/*",
     "https://outlook.office365.com/*",
     "https://outlook.office.com/*",
     "https://graph.microsoft.com/*",
     "https://login.microsoftonline.com/*",
-    "https://www.googleapis.com/*",
```

Comment:
Dangerous - this will likely cause all existing extensions to get disabled after update.

I forgot why we need this. Is it CORS? Is it to call them from content scripts?

Also needed if we relay requests through background page?

Reply:
@tomholub Sorry for the confusion @tomholub , the PR is still WIP. I'm actually not sure if this change is needed, I was planning to verify that after the verification from Google is done.

Reply:
Got it

Reply:
this change wasn't needed after all